src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -85,12 +85,11 @@ async function initializeConnection(options: BlazorOptions, logger: Logger): Pro
 
   connection.on('JS.BeginInvokeJS', DotNet.jsCallDispatcher.beginInvokeJSFromDotNet);
   connection.on('JS.EndInvokeDotNet', (args: string) => DotNet.jsCallDispatcher.endInvokeDotNetFromJS(...(JSON.parse(args) as [string, boolean, unknown])));
-  connection.on('JS.RenderBatch', (browserRendererId: number, batchId: number, batchData: Uint8Array) => {
-    logger.log(LogLevel.Debug, `Received render batch for ${browserRendererId} with id ${batchId} and ${batchData.byteLength} bytes.`);
 
-    const queue = RenderQueue.getOrCreateQueue(browserRendererId, logger);
-
-    queue.processBatch(batchId, batchData, connection);
+  const renderQueue = new RenderQueue(/* renderer ID unused with remote renderer */ 0, logger);
```

Comment:
Was this a bug? were we creating a new render queue for every batch?

Reply:
No, we had a cache of them by `browserRendererId` (that's what `RenderQueue.getOrCreateQueue` did).

The updated code doesn't need the cache because it doesn't have to address them by ID - it knows there's exactly one per circuit, and it already has a reference to it.

Reply:
OK dope. I'm, not super familar with what went on this area, thanks for indulging me 👍